lib/models/NationalStats.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+
+const schema = new mongoose.Schema({
+  totalNewNationalCases: {
+    type: Number
+  },
+  totalNewNationalDeaths: {
+    type: Number
+  },
+  totalNewNationalRecovered: {
+    type: Number
+  }
+}, { timestamps: true });
+
+module.exports = mongoose.model('Stat', schema);
```

Comment:
We may want to rename this model 'NationalStats' since we are talking about having a StateStats model.